### PR TITLE
Add support for SpinnerNoBlink to LegacyOldStyleSpinner

### PR DESCRIPTION
**Pull request by hbnrmx**

### CODE REVIEW

File: osu.Game.Rulesets.Osu/Skinning/LegacyOldStyleSpinner.cs [hunk 1]
```diff
@@ -25,12 +26,16 @@ public class LegacyOldStyleSpinner : CompositeDrawable
         private Sprite metreSprite;
         private Container metre;
 
+        private bool spinnerNoBlink;
```
Comment (smoogipoo) [inline osu.Game.Rulesets.Osu/Skinning/LegacyOldStyleSpinner.cs:29]: Given that this is always used in a `!spinnerNoBlink` manner, I'd like to see this variable inverted as `spinnerBlink`. You can do the inverse locally - don't need to change the name of the setting enum.

File: osu.Game/Skinning/LegacySkinConfiguration.cs [hunk 2]
```diff
@@ -19,6 +19,7 @@ public enum LegacySetting
             ComboOverlap,
             AnimationFramerate,
             LayeredHitSounds,
+            SpinnerNoBlink
```
Comment (smoogipoo) [inline osu.Game/Skinning/LegacySkinConfiguration.cs:22]: This should be in `OsuSkinConfiguration`, not `LegacySetting`.

**Review by smoogipoo — CHANGES_REQUESTED** [review]

File: osu.Game.Rulesets.Osu/Skinning/LegacyOldStyleSpinner.cs [hunk 3]
```diff
@@ -25,12 +25,16 @@ public class LegacyOldStyleSpinner : CompositeDrawable
...
 
+        private bool spinnerBlink;
+
         private const float sprite_scale = 1 / 1.6f;
         private const float final_metre_height = 692 * sprite_scale;
 
         [BackgroundDependencyLoader]
         private void load(ISkinSource source, DrawableHitObject drawableObject)
         {
+            spinnerBlink = !source.GetConfig<OsuSkinConfiguration, bool>(OsuSkinConfiguration.SpinnerNoBlink)?.Value ?? true;
```
Comment (bdach) [inline osu.Game.Rulesets.Osu/Skinning/LegacyOldStyleSpinner.cs:36]: Not sure if it's just a "me" problem, but this makes my brain hurt. I don't immediately know reading this if the default is `false` or `true` as I don't remember operator precedence by heart and I have to add braces to check (it does seem to be correct, however).

Comment (hbnrmx) [reply osu.Game.Rulesets.Osu/Skinning/LegacyOldStyleSpinner.cs]: Thanks! Added braces

**Review by smoogipoo — APPROVED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**